sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/EventProcessorSample.java

```diff
@@ -4,12 +4,11 @@
 package com.azure.messaging.eventhubs;
```

Comment:
One of the feedbacks was to have a reference sample of how we expect users implement handle errors (ie. if it is transient, or an AmqpException or etc). Is this in another PR?

Reply:
I will add that in another PR.